src/lib/core/DataModelTypes.h

```diff
@@ -39,13 +40,16 @@ typedef uint32_t EventId;
 typedef uint64_t EventNumber;
 typedef uint64_t FabricId;
 typedef uint8_t FabricIndex;
+typedef uint8_t SceneIndex;
 typedef uint32_t FieldId;
 typedef uint16_t ListIndex;
 typedef uint16_t LocalizedStringIdentifier;
 typedef uint32_t TransactionId;
 typedef uint16_t KeysetId;
 typedef uint8_t InteractionModelRevision;
 typedef uint32_t SubscriptionId;
+typedef GroupId SceneGroupID;
+typedef uint8_t SceneId;
```

Comment:
Please document how SceneId and SceneIndex differ.  And in particular, is SceneIndex a data model type at all?  I don't see it in the scenes cluster.